wgpu-hal/src/gles/adapter.rs

```diff
@@ -1144,15 +1143,6 @@ impl crate::Adapter<super::Api> for super::Adapter {
                 composite_alpha_modes: vec![wgt::CompositeAlphaMode::Opaque], //TODO
                 swap_chain_sizes: 2..=2,
                 current_extent: None,
-                extents: wgt::Extent3d {
-                    width: 4,
-                    height: 4,
-                    depth_or_array_layers: 1,
-                }..=wgt::Extent3d {
-                    width: self.shared.max_texture_size,
-                    height: self.shared.max_texture_size,
-                    depth_or_array_layers: 1,
-                },
```

Comment:
Could you just make sure there is validation on configure that the texture is within the device texture size.